force-app/main/default/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -1026,14 +1028,41 @@ public with sharing class CON_ContactMerge_CTRL {
         populateContactWithFields(winningContact, false);
     }
 
-    private void updateContactWithOnlyUniqueFields(Id winningContactId) {
+    private void updateContactAfterMerge(Id winningContactId) {
         Contact winningContact = new Contact(Id = winningContactId);
 
-        Boolean hasUniqueFieldChange = populateContactWithFields(winningContact, true);
+        Boolean hasFieldChange = populateContactWithOnlyUniqueFields(winningContact);
+        hasFieldChange = hasFieldChange || populateContactWithSustainerChanged(winningContact);
 
-        if (hasUniqueFieldChange) {
+        if (hasFieldChange) {
             updateAndHandleWinningContact(winningContact);
         }
+        
+    }
+
+    private Boolean populateContactWithOnlyUniqueFields(Contact winningContact) {
+        Boolean hasUniqueFieldChange = populateContactWithFields(winningContact, true);
+        return hasUniqueFieldChange;
+    }
+
+    private Boolean populateContactWithSustainerChanged(Contact winningContact) {
```

Comment:
I'd consider a new name for this method, `evaluateContactForSustainerChange`? Or since it actually makes the update, `evaluateAndUpdateContactForSustainerChange`?

Reply:
I like `evaluateContactForSustainerChange`. The method populates the field and returns true if the field is modified.